.prettierrc

```diff
@@ -3,7 +3,7 @@
   "semi": true,
   "singleQuote": true,
   "arrowParens": "avoid",
-  "printWidth": 120,
+  "printWidth": 85,
```

Comment:
Just think 120 - is too long for good readability/legibility of long function call chains